HHS/OPRE-OPS
# Ops 1316/task 1323

## What changed

- adds Generic 404 error page
- routes api errors here

Does not include: 
- update validation for Agreement type select
- add/update e2e tests

## Issue

#1323

## How to test

- create an agreement without selecting a type
- see new 404 error page
- with error Alert

## Screenshots

https://github.com/HHS/OPRE-OPS/assets/4629398/59acde1e-de75-4c59-b475-873b03b2c53e

## Links

- [uswds 404 page reference](https://designsystem.digital.gov/templates/404-page/)
